rho/authaddcommand.py

```diff
@@ -15,28 +15,50 @@
 """
 
 from __future__ import print_function
-import csv
+import json
 import os
 import sys
 import uuid
 from getpass import getpass
 from collections import OrderedDict
 from rho.clicommand import CliCommand
+from rho.vault import read_vault_password, get_vault_password, Vault
 from rho.translation import get_translation
 
 _ = get_translation()
 
 
-# Write new credentials in the related file.
-def _save_cred(cred):
-    append_or_write = 'a'
+def auth_exists(cred_list, auth_name):
+    """ Checks whether authentication credential already exists
+    :param cred_list: A list of credential dictionaries
+    :param auth_name: Name of credential to check for existence
+    :returns: True if auth_name exists, False otherwise
+    """
+    auth_found = False
+    for cred in cred_list:
+        if cred.get('name') == auth_name:
```

Comment:
No. A name is a required field for an auth credential.
https://github.com/quipucords/rho/blob/master/rho/authaddcommand.py#L78